cmd/admin-handlers.go

```diff
@@ -801,6 +810,91 @@ func convertValueType(elem []byte, jsonType gjson.Type) (interface{}, error) {
 	}
 }
 
+func isInvalidJSONField(expectedKeys gjson.Result, elem []byte) (string, string) {
```

Comment:
Add tests for this function.

Reply:
```
func checkValidJSONFields(expectedKeys gjson.Result, elem []byte) (error) 
```

Is a better function name, and `return nil` for success.

Reply:
OK